### ISSUE TITLE
Error response from daemon: manifest for mailu/roundcube:master not found: manifest unknown: manifest unknown

### ISSUE THREAD

**Issue by nike7o0** [Mailu/Mailu#2730]
## Environment & Version
Master
### Environment

- [x] docker compose
- [ ] kubernetes
- [ ] docker swarm

### Version

- Version: `master`

## Description
There is problem with roundcube, missing manifest error and the image cannot be pulled.

## Replication Steps
docker pull mailu/roundcube:master

## Observed behaviour

Error response from daemon: manifest for mailu/roundcube:master not found: manifest unknown: manifest unknown

## Expected behaviour
The image should be pullable.

## Logs

Error response from daemon: manifest for mailu/roundcube:master not found: manifest unknown: manifest unknown

**Comment by nike7o0** [comment]
Are you sure ? This command is not working at all - docker pull mailu/roundcube:master

**Comment by nextgens** [comment]
Yes I am sure. We have switched to a different registry and the new container name is webmail, not roundcube.

There are other significant changes that have been made.
